modules/awesomeBookApp.js

```diff
@@ -0,0 +1,49 @@
+import Book from './book.js';
+
+class AwesomeBookApp {
+  constructor() {
+    this.booksCollection = [];
+    this.addBookButton = document.getElementById('addBookBtn');
+
+    this.bookListDiv = document.getElementById('bookList');
+    this.displayBooks();
+  }
+
+  addBook(title, author) {
+    if (title !== '' && author !== '') {
+      const newBook = new Book(title, author);
+      this.booksCollection.push(newBook);
+
+      this.displayBooks();
+    }
+  }
```

Comment:
- [ ] Great job on implementing the addBook function! However, as per the project requirements, you also need to utilize local storage to store the books when adding a new book. Additionally, when deleting a book, you should remove it from the local storage as well. To achieve this, you can use the `localStorage` object in JavaScript to store and retrieve data. When adding a new book, you can convert the book object to a JSON string and then store it in the local storage using the `localStorage.setItem()` method. When deleting a book, you can remove it from the local storage using the `localStorage.removeItem()` method.

By adding this functionality, your project will fully meet the requirements and have a proper data storage mechanism using local storage. Keep up the good work! 👍📚🗄️